test/test_jit_cuda_fuser.py

```diff
@@ -570,7 +571,8 @@ def t(x: torch.Tensor, y: torch.Tensor):
             self.assertGraphContains(t_jit.graph_for(x, y), FUSION_GUARD)
         o = t(x, y)
         self.assertEqual(o.dtype, jit_o.dtype)
-        self.assertTrue(self._compare("failing case {}\n{}\n{}\n{}".format(dtype, operation, x, y), o, jit_o, 1e-2))
+        if o.dtype != torch.bfloat16:
```

Comment:
Are we not checking outputs for bfloat16?

Reply:
Yeah, bfloat16 is giving the following error on my system:
```
failing case torch.bfloat16
<built-in method exp of type object at 0x7fa559f971c0>
tensor([    -inf, -10.0000,  -3.1406,  -1.0000,  -0.5000,   0.0000,   1.0000,
          0.5000,   3.1406,  10.0000,      inf,      nan], device='cuda:0',
       dtype=torch.bfloat16)
tensor([0.4004, 0.2559, 0.0859, 0.1943, 0.6953, 0.6250, 0.8477, 0.2070, 0.7227,
        0.8320, 0.3809, 0.9922], device='cuda:0', dtype=torch.bfloat16) False
dif    :  tensor([[ -0.1250],
        [-48.0000]], device='cuda:0', dtype=torch.bfloat16)
inp1   :  tensor([[   9.6250],
        [4080.0000]], device='cuda:0', dtype=torch.bfloat16)
inp2   :  tensor([[   9.7500],
        [4128.0000]], device='cuda:0', dtype=torch.bfloat16)
maximum difference tensor(-0.1250, device='cuda:0', dtype=torch.bfloat16)
```
That is strange, because I didn't touch bfloat16 at all.

Reply:
Looks like this issue is fixed in https://github.com/csarofeen/pytorch/pull/1674